CoseSignTool.Tests/CoseSignTool.Tests.csproj

```diff
@@ -24,19 +24,32 @@
 	</ItemGroup>
 
 	<!-- Package references -->
+	<ItemGroup>

```

Comment:
If every test project uses these, perhaps we should move them to CoseSign1.Tests.Common?